akka-docs/src/main/paradox/typed/routers.md

```diff
@@ -88,4 +88,4 @@ it will not give better performance to create more routees than there are thread
 
 Since the router itself is an actor and has a mailbox this means that messages are routed sequentially to the routees
 where it can be processed in parallel (depending on the available threads in the dispatcher).
-In a high throughput use cases the sequential routing could be a bottle neck. Akka Typed does not provide an optimized tool for this.
+In a high throughput use cases the sequential routing could be a bottle neck. Akka Typed does not provide an optimized tool for this.
```

Comment:
Is there a notion of yet here?

Reply:
You mean as in "there isn't yet"? We don't plan to introduce one to not end up with the complicated internals we have for untyped to support that.